agreement/common_test.go

```diff
@@ -214,6 +216,28 @@ func makeTestLedger(state map[basics.Address]basics.BalanceRecord) Ledger {
 	}
 
 	l.notifications = make(map[basics.Round]signal)
+
+	l.consensusVersion = func (r basics.Round) (protocol.ConsensusVersion, error) {
+		return protocol.ConsensusCurrentVersion, nil
+	}
+	return l
+}
+
+func makeTestLedgerWithConsensusVersion(state map[basics.Address]basics.BalanceRecord, consensusVersion func(basics.Round) (protocol.ConsensusVersion, error)) Ledger {
+	l := new(testLedger)
+	l.entries = make(map[basics.Round]bookkeeping.Block)
+	l.certs = make(map[basics.Round]Certificate)
+	l.nextRound = 1
+
+	// deep copy of state
```

Comment:
I don't think you introduced this comment, but we should toss it, since it's now misleading.

Reply:
should this be tossed in the original? aka line 212?

Reply:
Yeah--let's remove it there as well.